src/main/webapp/app/shared/components/code-button/code-button.component.ts

```diff
@@ -139,6 +144,7 @@ export class CodeButtonComponent implements OnInit, OnChanges {
      */
     private addCredentialsToHttpUrl(url: string, insertPlaceholder = false): string {
         const includeToken = this.versionControlAccessTokenRequired && this.user.vcsAccessToken;
+        console.log(includeToken);
```

Comment:
Don't forget to remove logging later :) Or shall that stay?

Reply:
I removed it :+1: was left from debugging